src/SFA.DAS.EAS.Api/Orchestrators/AccountsOrchestrator.cs

```diff
@@ -91,8 +91,8 @@ public AccountsOrchestrator(IMediator mediator, ILog logger, IMapper mapper, IHa
         public async Task<OrchestratorResponse<LegalEntityViewModel>> GetLegalEntity(string hashedAccountId, long legalEntityId)
         {
             _logger.Info($"Getting legal entity {legalEntityId} for account {hashedAccountId}");
-
-            var legalEntityResult = await _mediator.SendAsync(new GetLegalEntityByIdQuery { Id = legalEntityId });
+            var accountId = _hashingService.DecodeValue(hashedAccountId);
+            var legalEntityResult = await _mediator.SendAsync(new GetLegalEntityByIdQuery { AccountId = accountId, Id = legalEntityId });

```

Comment:
Could we pass the hashedAccountId to the Query and decode the value in there?  That'd be more consistent with the other queries.